src/status_im2/contexts/shell/jump_to/components/bottom_tabs/view.cljs

```diff
@@ -1,10 +1,11 @@
 (ns status-im2.contexts.shell.jump-to.components.bottom-tabs.view
-  (:require [utils.re-frame :as rf]
+  (:require [quo2.core :as quo]
+            [quo2.theme :as theme]
```

Comment:
nit import as quo.theme to avoid clashes with `theme` prop - (perhaps it won't matter in this file though 🤔 )